### ISSUE TITLE
Sign/submit the extension ZIP without building

### ISSUE THREAD

**Issue by filips123** [mozilla/web-ext#3242]
### Is this a feature request or a bug?

Feature request.

### What is the current behavior?

`web-ext sign` always [builds the extension](https://github.com/mozilla/web-ext/blob/master/src/cmd/sign.js#L57-L63) (packages it into ZIP), and there is no way to sign/submit an existing extension ZIP file.

This makes it harder to use it when you've already previously built and packaged the extension with another tool, and only want to submit the packaged ZIP file (and the source code ZIP file).

In my case, the extension is built using another tool (and then packaged into ZIP using `web-ext build`, but this doesn't really matter) on GitHub Actions, and both the source code and the compiled extension ZIPs are uploaded as an artifacts. Then, in another workflow, I want to just download these two files and submit them to AMO using `web-ext-sign`. However, because `web-ext sign` expects a directory and always builds the extension, I would have to unzip the already-packaged extension manually and let `web-ext sign` package it itself. This seems unnecessary and may also cause problems if I was packaging the extension with other tool than `web-ext build`, as different files could be packaged.

### What is the expected or desired behavior?

It should be possible to run a command that accepts a path to the already-packaged ZIP file of the extension (in addition to the source code ZIP file and the AMO metadata JSON file), verifies it, and directly publishes it to AMO.

This is similar to #3241, but that issue is about programmatically signing an extension file without building it, while my issue is about adding a command (or argument to `web-ext sign`) to do that.

**Comment by Rob--W** [comment]
Out of curiosity, because I noticed that these two related feature requests were filed close to each other: was there any recent event or publication that is causing an increased demand for sign-zip file functionality?

**Comment by filips123** [comment]
Not really. I wanted to set up automatic building and publishing process for my extension for some some, but didn't have much time until now. Then I've noticed that `web-ext` currently doesn't support submitting ZIP files (which I already mostly set up before) and opened this issue. So it's just a coincidence.

**Comment by kewisch** [comment]
In my case I was working out dependency updates to to https://github.com/kewisch/action-web-ext/ and saw sign-addon is deprecated, so no relation to a recent event. @filips123 you might want to look into that action if you want to automate things, it can currently sign a previously existing zip file.

**Comment by kewisch** [comment]
Fixing this issue would resolve 3241 for me as well though, I could simply call webExt.cmd.sign with a zip file and it would trigger signing. Whatever works best for you Rob :)
